tests/lib/cli-engine/cli-engine.js

```diff
@@ -3541,36 +3664,54 @@ describe("CLIEngine", () => {
         describe("glob pattern '[ab].js'", () => {
             const root = getFixturePath("cli-engine/unmatched-glob");
 
-            it("should match '[ab].js' if existed.", () => {
-                CLIEngine = defineCLIEngineWithInMemoryFileSystem({
-                    cwd: () => root,
+            /* eslint-disable-next-line func-style */
+            let cleanup = () => {};
+
+            afterEach(async () => {
+                await cleanup();
+                cleanup = () => {};
+            });
```

Comment:
This is one of a few other places that could be updated to the [refactor @mdjermanovic suggested](https://github.com/eslint/eslint/pull/13709#discussion_r495894365). Did you want to do that? Looks like a search for `afterEach(async` in the file should find the rest if you decide to change them.